www/js/app.js

```diff
@@ -10,129 +10,133 @@ angular.module('pvta', ['ionic', 'ngCordova', 'pvta.controllers', 'angularMoment
 
 .run(function ($ionicPlatform) {
   $ionicPlatform.ready(function () {
-    // Hide the accessory bar by default (remove this to show the accessory bar above the keyboard
-    // for form inputs)
+                // Hide the accessory bar by default (remove this to show the accessory bar above the keyboard
+                // for form inputs)
     if (window.cordova && window.cordova.plugins.Keyboard) {
       cordova.plugins.Keyboard.hideKeyboardAccessoryBar(true);
       cordova.plugins.Keyboard.disableScroll(true);
     }
   });
-  /******************************************************
-  * Set the status bar color to our app's color.
-  *********************************************************
-  * Only after the device has specifically told us it's ready
-  * may we access the global StatusBar object.
-  * **************************************************/
+        /******************************************************
+         * Set the status bar color to our app's color.
+         *********************************************************
+         * Only after the device has specifically told us it's ready
+         * may we access the global StatusBar object.
+         * **************************************************/
   document.addEventListener('deviceready', onDeviceReady, true);
   function onDeviceReady () {
     StatusBar.backgroundColorByHexString('#387ef5');
   }
 })
 
-.config(function ($stateProvider, $urlRouterProvider, $ionicFilterBarConfigProvider) {
```

Comment:
Curious to know the reason behind the removal of the `$ionicFilterBarConfigProvider`. Is it unnecessary?